diff --git a/dist/spark.cjs.js b/dist/spark.cjs.js
index 4ab8da5494a00c820eb75af186e81d46a4563eec..ba55b29878e410568395076655dc50a0fd994788 100644
--- a/dist/spark.cjs.js
+++ b/dist/spark.cjs.js
@@ -16789,7 +16789,7 @@ function restoreHidden(hidden) {
   }
 }
 function isRenderableObject(object) {
-  return object instanceof THREE__namespace.Mesh || object instanceof THREE__namespace.Line || object instanceof THREE__namespace.Points || object instanceof THREE__namespace.Sprite;
+  return object.isMesh === true || object.isSkinnedMesh === true || object.isInstancedMesh === true || object.isLine === true || object.isPoints === true || object.isSprite === true;
 }
 function getObjectMaterials(object) {
   const material = object.material;
diff --git a/dist/spark.module.js b/dist/spark.module.js
index 93a778fc0aae9f4f2599635ce2fe890e84413cbc..f6ec3821f3b0ed82b60333816007e17d1bb6cb4e 100644
--- a/dist/spark.module.js
+++ b/dist/spark.module.js
@@ -16770,7 +16770,7 @@ function restoreHidden(hidden) {
   }
 }
 function isRenderableObject(object) {
-  return object instanceof THREE.Mesh || object instanceof THREE.Line || object instanceof THREE.Points || object instanceof THREE.Sprite;
+  return object.isMesh === true || object.isSkinnedMesh === true || object.isInstancedMesh === true || object.isLine === true || object.isPoints === true || object.isSprite === true;
 }
 function getObjectMaterials(object) {
   const material = object.material;
